### ISSUE TITLE
Can bot creat a game without say"next game is xxx"

### ISSUE THREAD

**Issue by theboyxucool** [kolton/d2bot-with-kolbot#1264]

**Comment by DarkHorseDre** [comment]
Yes, check the character config file and all associated comments:

Config.BaalAssistant.NextGameMessage = ["Next Game", "Next", "New Game"];	// Next Game message, this is a precautionary quit command, Recommended setting up: Config.QuitList

There are also other messages defined within the configs

**Comment by theboyxucool** [comment]
thanks, that,s work,s well
